### DISCUSSION TITLE
Can this application cause ban in Valorant game?

### DISCUSSION THREAD

**Discussion by mindstop** [maximmax42/Discord-CustomRP#113] [General]
I recently got banned in valorant for usage of 3rd party application and it was a permanent ban. I did not use any cheats but since they said any 3rd party app can cause ban, wanted to check if this application can cause ban. 
![image](https://user-images.githubusercontent.com/76905913/160293636-6cc4d87e-b737-443e-9ee3-07506860ee7e.png)

**Comment by maximmax42** [comment]
I don't play Valorant so I have no idea. It's not likely, but who knows.

If anyone else plays Valorant and is using CustomRP please message here.
